components/brave_wallet/browser/brave_wallet_p3a.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
```

Comment:
2022

Reply:
Updated in https://github.com/brave/brave-core/pull/11929/commits/3432f0ad7540df051620f5d3c130386352502d4f